vmaas/types.go

```diff
@@ -191,13 +193,14 @@ type SRPMPkgNamesRequest struct {
 }
 
 type Update struct {
-	Package     string `json:"package"`
-	PackageName string `json:"package_name"`
-	EVRA        string `json:"evra"`
-	Erratum     string `json:"erratum"`
-	Repository  string `json:"repository"`
-	Basearch    string `json:"basearch"`
-	Releasever  string `json:"releasever"`
+	Package      string `json:"package"`
+	PackageName  string `json:"package_name"`
+	EVRA         string `json:"evra"`
+	Erratum      string `json:"erratum"`
+	Repository   string `json:"repository"`
+	Basearch     string `json:"basearch"`
+	Releasever   string `json:"releasever"`
+	Organization string `json:"organization"`
```

Comment:
do we want to always return `Organization` in Updates response, even when we are using hosted vmaas? Shouldn't be Organization returned also for Vulnerabilities response?

Reply:
I've added it to the Updates API response to identify the repo where the updated package is (since it's now possible to request Updates API without organization string and then it returns results from all organizations). In Vulnerabilities response there is no package/repo reference so it's not needed (except if we returned as a meta information).

I wouldn't differentiate hosted response somehow but we can consider forcing Updates API to return results always only from a single org. In this case the Organization in response wouldn't be needed at all.

Reply:
yes that's my point Organization is not needed in response for hosted VMaaS where it has no meaning and might be confusing for external users

Reply:
Removed Organization from Updates response, now 'DEFAULT' is assumed when org is not in the request.